docs/reference/migration/migrate_8_0/search.asciidoc

```diff
@@ -50,4 +50,12 @@ The vector functions of the form `function(query, doc['field'])` were
 deprecated in 7.6, and are now removed in 8.x. The form
 `function(query, 'field')` should be used instead. For example,
 `cosineSimilarity(query, doc['field'])` is replaced by
-`cosineSimilarity(query, 'field')`.
\ No newline at end of file
+`cosineSimilarity(query, 'field')`.
+
+[float]
+==== Consistent rounding of range queries on `date_range` fields
+`range` queries on `date_range` field currently can have slightly differently boundaries than their equivalent
+query on a pure `date` field. This can e.g. happen when using date math or dates that don't specify up to the
+last millisecond. While queries on `date` field round up to the latest millisecond for `gt` and `lte` boundaries,
+the same queries on `date_range` fields didn't do this. The behavior is now the same for both field types like
+documented in <<range-query-date-math-rounding>>.
```

Comment:
I think we should skip this change on master and only have it in the 7.7 migration guide on the 7.x branch?